bcgov/moh-prime
# Feature/prime 2169 filter renewal admin

To Test:

- Add Enrolment that is manually reviewed
- Add another enrolment that is automatically reviewed, Then resubmit it as a manual adjudicated (obo to auto, then add cert)
- Add another enrolment that is manually adjudicated, approve and accept a toa, then resubmit to trigger manual again

Once those are in that state go into the admin view of enrollees and add the new filter, you should only see enrolments that have been previously manually adjudicated and are in the UNDER_REVIEW status
